src/index.js

```diff
@@ -4,52 +4,70 @@ import ReactDOMServer from 'react-dom/server';
 import hypernova, { serialize, load, toScript, fromScript } from 'hypernova';
 import { StyleSheet, StyleSheetServer } from 'aphrodite';
 
-export const renderReactWithAphrodite = (name, component) => hypernova({
-  server() {
-    return (props) => {
-      const { html, css } = StyleSheetServer.renderStatic(() => (
-        ReactDOMServer.renderToString(React.createElement(component, props))
-      ));
-
-      const style = `<style data-aphrodite="data-aphrodite">${css.content}</style>`;
-      const markup = serialize(name, html, props);
-      const classNames = toScript({ 'aphrodite-css': name }, css.renderedClassNames);
-
-      return `${style}\n${markup}\n${classNames}`;
-    };
-  },
-
-  client() {
-    const classNames = fromScript({ 'aphrodite-css': name });
-    if (classNames) StyleSheet.rehydrate(classNames);
-
-    const payloads = load(name);
-    if (payloads) {
-      payloads.forEach((payload) => {
-        const { node, data } = payload;
-        if (node) {
-          const element = React.createElement(component, data);
-          ReactDOM.render(element, node);
-        }
-      });
-    }
-
-    return component;
-  },
-});
-
-export const renderReactWithAphroditeStatic = (name, component) => hypernova({
-  server() {
-    return (props) => {
-      const { html, css } = StyleSheetServer.renderStatic(() => (
-        ReactDOMServer.renderToStaticMarkup(React.createElement(component, props))
-      ));
-
-      const style = `<style data-aphrodite="data-aphrodite">${css.content}</style>`;
-
-      return `${style}\n${html}`;
-    };
-  },
-
-  client() {},
-});
+const config = {
+  enhancers: [], // HOCs applied to the component before render: e0(e1(e2(...en(component))))
```

Comment:
Will this be a problem that it's a global?

Reply:
hmm, it very well might be; we'd want a way to clear it across requests